auth/exported-token.service.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function ExportedTokenService (options) {
```

Comment:
We should remove all the code concerning token services, instead of creating this dummy token service, as this is not the responsibility of api-particulier anymore.